internal/scorecard/alpha/formatting.go

```diff
@@ -42,17 +42,17 @@ func (r PodTestRunner) getTestStatus(ctx context.Context, p *v1.Pod) (output *v1
 // run based on user selection
 func (o Scorecard) List() v1alpha3.TestList {
 	output := v1alpha3.NewTestList()
-	tests := o.selectTests()
-
-	for _, test := range tests {
-		item := v1alpha3.NewTest()
-		item.Spec = v1alpha3.TestSpec{
-			Image:      test.Image,
-			Entrypoint: test.Entrypoint,
-			Labels:     test.Labels,
+	for _, stage := range o.Config.Stages {
```

Comment:
Is it possible to have duplicate tests across stages? I suspect this wouldn't be common if so.

Reply:
As far as I know, there is nothing that would prevent it. Scorecard would happily run it twice and report the results twice. It would probably also handle duplicate tests in a single stage the same way.

Reply:
Running duplicates is obviously fine. Test names listed may be duplicates though, which looks a little weird.